kitchen/node/Cargo.toml

```diff
@@ -38,6 +38,8 @@ basic-authorship = { package = "substrate-basic-authorship", git = 'https://gith
 # Your runtime must have the necessary runtime modules to support consensus (Babe, Grandpa, etc)
 runtime = { package = "super-runtime", path = "../runtimes/super-runtime" }
 runtime-genesis = { package = "super-genesis", path = "../runtimes/super-genesis" }
+# runtime = { package = "weight-fee-runtime", path = "../runtimes/weight-fee-runtime"}
```

Comment:
```suggestion

```

Reply:
The idea is that user/learner can edit this section of the node to experiment with different runtimes. By having a few available there, they can easily see what to type. Do you think it's better not to have the second example there? If so, then definitely the following line importing `runtime-genesis` should go as well.

Reply:
I think it's better to leave it as commented out. The instructions for using this or `runtime-genesis` should mention uncommenting these lines instead of copying and pasting this import path. I don't see any gain from asking the user to copy and past the import path so might as well keep things as easy as possible.

I also wonder if this will make it easier to test everything with the CI. I'm going to take a look at that PR next.